src/components/DataInput/DataInput.cy.tsx

```diff
@@ -0,0 +1,164 @@
+import React from "react";
+import CheckboxInput from "@/components/DataInput/CheckboxInput";
+import DropdownListInput from "@/components/DataInput/DropdownListInput";
+import FreeFormTextInput from "@/components/DataInput/FreeFormTextInput";
+import RadioGroupInput from "@/components/DataInput/RadioGroupInput";
+import {
+    getFreeFormTextHandler,
+    getCheckboxHandler,
+    getDropdownListHandler,
+    getRadioGroupHandler,
+} from "@/components/DataInput/inputHandlerFactories";
+import { SelectChangeEvent } from "@mui/material";
+
+describe("Data Input Components", () => {
+    it("renders", () => {
+        cy.mount(
+            <CheckboxInput
+                label="LABEL"
+                onChange={getCheckboxHandler(() => console.log("Checkbox Changed"))}
+            />
+        );
+        cy.mount(
+            <DropdownListInput
+                labelsAndValues={[
```

Comment:
I think this has been used in many of the individual tests. Would it be better to make it a const before all the tests?